LiveStream/ViewModels/LiveVideoViewModel.swift

```diff
@@ -42,6 +48,15 @@ internal protocol LiveVideoViewModelOutputs {
 
   /// Emits a stream when the subscriber of that stream should be removed.
   var removeSubscriber: Signal<OTStreamType, NoError> { get }
+
+  /// Emits when all subscribers should be re-subscribed when the view reappears.
+  var resubscribeAllSubscribersToSession: Signal<(), NoError> { get }
+
+  /// Emits to toggle play/pause when the view disappears/reappears.
+  var setHlsToPlaybackState: Signal<Bool, NoError> { get }
```

Comment:
I flipped the booleans so this now sets it to play in which case `setHlsToPlaybackState` kind've makes sense?